Makefile

```diff
@@ -27,16 +27,19 @@
 
 .EXPORT_ALL_VARIABLES:
 
-ENV_DIR  := .env
+ENV_DIR  := .venv
 ENV_BIN  := $(ENV_DIR)/bin
 ACTIVATE := $(ENV_BIN)/activate
 PYTHON   := python3
 PIP      := $(ENV_BIN)/pip3
 TWINE    := $(ENV_BIN)/twine
+NODE     := $(ENV_BIN)/node
 NODEENV  := $(ENV_BIN)/nodeenv
 DIST_DIR := .dist
 PYPIRC   := ~/.pypirc
 
+NODEJS_VERSION := 16.18.1
```

Comment:
Does anything speak against using the latest LTS version 18?

Reply:
Good idea. Let's do it on behalf of another PR, #365, in order to be able to check if that will also introduce some slowness on CI, like observed at https://github.com/crate/crate-admin/pull/797.